src/main/kotlin/no/nav/modiapersonoversikt/skrivestotte/service/LeaderElectorService.kt

```diff
@@ -35,8 +36,8 @@ class LeaderElectorService(val configuration: Configuration) {
     internal fun getLeader(): LeaderElectorResponse {
         return try {
             runBlocking {
-                val response = client.get(configuration.electorPath)
-                response.bodyAsText().fromJson()
+                val response = client.get(configuration.electorPath).body<String>()
+                response.fromJson()
```

Comment:
Dette tror jeg ikke vil løse problemet med `Could not get leader from http://localhost:4040`. Men det er enklere å se om vi splitter opp koden litt.

**Før:**
```kotlin
val response = client.get(configuration.electorPath)
val body: String = response.bodyAsText()
return body.fromJson() // Typen inferert av kotlin
```

**Etter:**
```kotlin
val response = client.get(configuration.electorPath)
val body: String = response.body<String>()
return body.fromJson() // Typen inferert av kotlin
```
Siden feilen skjer som en følge av kallet til `fromJson` så vil endringen derfor ikke ha noen effekt på feilen vi ser i preprod.